docfx/docs/migratingfromv1.md

```diff
@@ -175,6 +176,12 @@ The API for handling keyboard input is significantly improved. See [Keyboard API
 * Use [View.Keybindings](~/api/Terminal.Gui.View.Keybindings.yml) to configure key bindings to `Command`s.
 * It should be very uncommon for v2 code to override `OnKeyPressed` etc... 
 * Anywhere `Ctrl+Q` was hard-coded as the "quit key", replace with `Application.QuitKey`.
+* Replace `Application.RootKeyEvent` with `Application.KeyDown`.  
```

Comment:
Please add to the end of this bullet: "If the reason for subscribing to `RootKeyEvent` was to enable an application-wide action based on a key-press, consider using `Application.KeyBindings` instead.